Add task status (#24)
flyve-mdm/doc: source/getting-started/index.rst

Once you assign the devices to the Fleet, all the policies will be immediately applied!

Task Status
~~~~~~~~~~~

You can check the status of the policies deployed.

============ ===============================
Status       Description
------------ -------------------------------

> Pending      Waiting for implementation

Reviewer: ```suggestion
Pending      Waiting for propagation
```

> Received 

Reviewer: ```suggestion
Received    Propagated to the device
```

> Done

Reviewer: ```suggestion
Done    Task successfully applied 
```

> Failed

Reviewer: ```suggestion
Failed   The task could not apply
```

> Canceled

Reviewer: ```suggestion
Canceled   The task was canceled 
```

> Incompatible

Reviewer: ```suggestion
Incompatible   The task cannot apply on the device because it is incompatible
```

> Overriden

Reviewer: ```suggestion
Overriden     The task is overiden by a similar one applied on the device itself```

> Waiting

Reviewer: ```suggestion
Waiting    The task is received by the device and is not applied yet
```

Once you assign the devices to the Fleet, all the policies will be immediately applied!

Task Status
~~~~~~~~~~~

You can check the status of the policies deployed.

============  =========================================================
Status        Description
------------  ---------------------------------------------------------

> Pending       Waiting for implementation, not yet received by the Agent

Reviewer: ```suggestion
Pending       Waiting for publication of the task, not yet received by the Agent
```

> Received      Succesfully received by the Agent, not yet implemented

Reviewer: ```suggestion
Received      Succesfully received by the Agent, not yet applied
```

> Done          Succesfully implemented

Reviewer: ```suggestion
Done          Succesfully applied
```

> Failed        Policy could not be implemented

Reviewer: ```suggestion
Failed        Policy could not be applied due to a failure
```

Canceled      Policy deleted or value changed
Incompatible  Policy not available for device
Overriden     If a fleet and Agent's policy conflict, the fleet policy is overriden prevailing the Agent's

> Waiting       Only for deploy policies which require time for implementation

Reviewer: ```suggestion
Waiting       Only for deploy policies which require time for application
```
